multirun/def.bzl

```diff
@@ -53,7 +53,7 @@ def _multirun_impl(ctx):
             runfiles = runfiles.merge(default_runfiles)
         commands.append(struct(
             tag = tag,
-            path = exe.short_path,
+            path = "./" + exe.short_path,
```

Comment:
I think a better approach would be to resolve relative paths in the instructions file taking the directory of that file as the base.

Reply:
I've tried this but it's not that straightforward because the path of the instruction file is relative to the exec root (or whatever) and it's just not worth the effort to do this.